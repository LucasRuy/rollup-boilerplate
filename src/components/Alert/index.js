import React from 'react'
import PropTypes from 'prop-types'
import styled from 'styled-components'

const AlertContainer = styled.span`
  padding: 10px;
  text-align: center;
`

const Alert = ({ message }) => <AlertContainer>{message}</AlertContainer>

Alert.defaultProps = {
  message: 'This is default prop.'
}

Alert.propTypes = {
  message: PropTypes.string
}

export default Alert
