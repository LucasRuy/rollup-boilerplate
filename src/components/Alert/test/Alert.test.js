import React from 'react'
import renderer from 'react-test-renderer'

import Alert from './../../Alert'

describe('<Alert />', () => {
  it('render correctly', () => {
    const alert = renderer.create(<Alert />).toJSON()

    expect(alert).toMatchSnapshot()
  })
})
