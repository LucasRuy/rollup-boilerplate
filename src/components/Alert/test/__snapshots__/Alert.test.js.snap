// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Alert /> render correctly 1`] = `
<span
  className="sc-bdVaJa hYmdNH"
>
  This is default prop.
</span>
`;
